asdf_astropy/extensions.py

```diff
@@ -443,14 +449,17 @@
     converters=ASTROPY_CONVERTERS,
 )
 
-# unit and quantity are part of the ASDF Standard core tags,
+# These tags are part of the ASDF Standard,
```

Comment:
I regret allowing FITS stuff in the basic standard. Sorry for the digression. Maybe we can sneak it out one day.

Reply:
I wonder if anyone even needs to store FITS in the tree in a structured way.  Maybe the idea to allow embedding general files in blocks would serve FITS just as well (see email chain "Embedding general files in ASDF" from 2021-01-28).